app/index.php

```diff
@@ -13,39 +14,57 @@
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+            integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+            crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+            integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+            crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+            integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+            crossorigin="anonymous"></script>
 </head>
 <body>
+<div>
+</div>
 <nav class="nav nav-pills flex-column flex-sm-row">
     <a class="col-2 flex-sm-fill text-sm-center nav-link" href="#">Recipe Finder</a>
     <a id="title" class="col-8 flex-sm-fill text-sm-center nav-link align-middle" href="#">Bread and Ifits.</a>
-    <a class="col-2 flex-sm-fill text-sm-center nav-link" href="#" tabindex="-1" aria-disabled="true">Go to recipe
-        book</a>
 </nav>
 <div class="nav-transform"></div>
 <div class="container-fluid">
     <div class="row" id="mainContent">
-        <aside class="col-2 justify-content-center">
-            <form method="post" id="ingredientsForm">
-                <?php
-                $db = DbConnector::getDatabase();
-                $ingredients = IngredientHydrator::getIngredients($db);
-                echo  DisplayIngredients::displayIngredients($ingredients);
-                ?>
-            </form>
-            <input class="choice-btn col-2" id="getRecipeBtn" type="submit" value="Get Recipe">
-        </aside>
+        <div class="col-2">
+            <aside class="justify-content-center">
+                <form method="post" id="ingredientsForm">
+                    <?php
+                    $db = DbConnector::getDatabase();
+                    $ingredients = IngredientHydrator::getIngredients($db);
+                    echo DisplayIngredients::displayIngredients($ingredients);
+                    $validator = IngredientValidator::checkUserInput($_POST);
+                    if ($validator) {
```

Comment:
you could just run the method in the if and not have a variable at all

Reply:
changed the above to reflect this